### PR TITLE
EDUCATOR-4889: Confirm deletion for learner uploaded files

### PR DESCRIPTION
**Pull request by jansenk**
[EDUCATOR-4889: Confirm deletion for learner uploaded files](https://openedx.atlassian.net/browse/EDUCATOR-4889)

add a confirmation step before deleting a file
I also added a django admin for SharedFileUploads because I ran into the absence of one while trying to manually test

### Popup
Screen Shot 2020-02-20 at 1 03 58 PM

### Clicked cancel, no deletion
Screen Shot 2020-02-19 at 5 01 43 PM

### Clicked ok, deleted
Screen Shot 2020-02-19 at 5 01 51 PM

### CODE REVIEW

File: openassessment/xblock/static/js/spec/lms/oa_response.js [hunk 1]
```diff
@@ -894,6 +896,31 @@ describe("OpenAssessment.ResponseView", function() {
...
+        expect(view.submitEnabled()).toBe(false);
+
+        // Upload file
+        var files = [{type: 'image/jpeg', size: 1024, name: 'picture1.jpg', data: ''}];
+        view.prepareUpload(files, 'image', ['i1']);
+        view.uploadFiles()
+        view.checkSubmissionAbility(true);
+        expect(view.submitEnabled()).toBe(true);
+
+        // Delete the first file, twice, but don't
```
Comment (nsprenkle) [inline openassessment/xblock/static/js/spec/lms/oa_response.js]: nit: what does this comment mean?

File: openassessment/assessment/admin.py [hunk 2]
```diff
@@ -149,7 +151,22 @@ def assessments_by(self, assessment_feedback):
     assessments_by.allow_tags = True
 
 
+class SharedFileUploadAdmin(admin.ModelAdmin):
+    """
+    Django admin model for PeerWorkflows.
```
Comment (iloveagent57) [inline openassessment/assessment/admin.py]: Copypasta

Comment (jansenk) [reply openassessment/assessment/admin.py]: grr

File: openassessment/xblock/static/js/src/lms/oa_response.js [hunk 3]
```diff
@@ -786,14 +786,24 @@ OpenAssessment.ResponseView.prototype = {
...
+                block.prop('deleted', true);
+                view.checkSubmissionAbility();
+            }).fail(function(errMsg) {
+                view.baseView.toggleActionError('delete', errMsg);
+            });
+        });
+    },
+
+    confirmRemoveUploadedFile: function() {
+        var msg = gettext('Are you sure you want to delete this file? It cannot be restored.');
```
[the remark this answers is not in the document]
Comment (jansenk) [reply openassessment/xblock/static/js/src/lms/oa_response.js]: I can grab the text that we have displayed on the page for the file fairly easily, like  '<file_description> (<filename>)'

**Comment by jansenk** [comment]
Updated description with new alert text

**Pull request merged**